ml-algorithms/src/test/java/org/opensearch/ml/engine/MLEngineTest.java

```diff
@@ -68,6 +70,22 @@ public void testPrebuiltModelPath() {
         assertEquals("https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/msmarco-distilbert-base-tas-b/1.0.1/torch_script/config.json", prebuiltModelConfigPath);
     }
 
+    @Test
+    public void testDeployModelZipPath() {
+        String modelId = "test_id";
+        String modelName = "huggingface/sentence-transformers/msmarco-distilbert-base-tas-b";
+        String modelZipPath = mlEngine.getDeployModelZipPath(modelId, modelName);
+        assertEquals(mlEngine.getMlCachePath() + "/models_cache/deploy/test_id/huggingface/sentence-transformers/msmarco-distilbert-base-tas-b.zip", modelZipPath);
+    }
+
+    @Test
+    public void testGetDeployModelChunkPath() {
+        String modelId = "test_id";
+        Integer chunkNum = 1;
```

Comment:
@dhrubo-os is the number of chunks hard coded to be always 10?  Wondering if we should have a loop here to test all of them.

Reply:
> @dhrubo-os is the number of chunks hard coded to be always 10? Wondering if we should have a loop here to test all of them.

@austintlee Thanks for the feedback! I added a loop here to check for chunks between 1 and 10.